task/rhacs-generic/0.1/README.md

```diff
@@ -0,0 +1,110 @@
+# Red Hat Advanced Cluster Security generic Task
+
+Allows users to customize and extend roxctl command line based on their needs.
+
+This task first exchanges a service account token against a short-lived RHACS
+authorization token, then performs the requested action.
+
+**Note: this Task requires a 4.4.2 roxctl image (task default) or a more recent
+image version.**
+
+## Prerequisites
+
+This task requires an active installation of [Red Hat Advanced Cluster Security (RHACS)](https://www.redhat.com/en/resources/advanced-cluster-security-for-kubernetes-datasheet)
+or [StackRox](https://www.stackrox.io).  It also requires configuration of a
+machine-to-machine integration.
+
+<https://www.redhat.com/en/technologies/cloud-computing/openshift/advanced-cluster-security-kubernetes>
+
+## Install the Task
+
+```bash
+kubectl apply -f https://api.hub.tekton.dev/v1/resource/tekton/task/rhacs-generic/0.1/raw
+```
+
+## Parameters
+
+- **`rox_central_endpoint`**: The address:port tuple for StackRox Central.
+Default: **central.stackrox.svc:443**
+- **`insecure-skip-tls-verify`**: Skip verification the TLS certs of the Central
+endpoint and registry. Examples: _"true", **"false"**_.
+- **`rox_arguments`**: The command line that you would like to enter, in array
+form. (required)
+- `rox_image`: The image providing the roxctl tool (optional).
+Default: quay.io/stackrox-io/roxctl:4.4.2
+(this is also the minimum version working with this task). 
+- `output_file`: path to a file where to redirect roxctl standard output.
+Default: "" (redirects to stdout).
+- `error_file`: path to a file where to redirect roxctl standard error.
+Default: "" (redirects to stderr).
+
+## Workspaces
+
+- **data**: An [optional Workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#optional-workspaces)
+which stores files used as parameters to the command line.
+
+## Usage
+
+Check the [documentation](https://docs.openshift.com/acs/operating/manage-user-access/configure-short-lived-access.html#configure-short-lived-access_configure-short-lived-access)
+to configure the trust with the OIDC token issuer. This
+[example](../../rhacs-m2m-authenticate/0.1/samples/configure-m2m.md) describes
+a possible RHACS machine-to-machine integration configuration.
+
+**Example task uses:**
+
+Deployment check:
```

Comment:
Let's provide examples which are _not_ the deployment check / image check examples since for those we'd rather have folks use the appropriate tasks.

We can use the netpol command or the declarative config command as an example, and then refer to this [section in the doc](https://docs.openshift.com/acs/4.4/cli/command-reference/roxctl.html) for an overview of all roxctl commands.